sdk/containerregistry/azure-containerregistry/CHANGELOG.md

```diff
@@ -5,8 +5,10 @@
 ### Features Added
 
 ### Breaking Changes
+- The enum properties in `ArtifactManifestProperties` can be `string` when the value is not included in enum.
```

Comment:
Could you give more information why this is breaking?

Reply:
Removed this line as the changes are no longer causing breaking changes.